src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -5565,6 +5676,20 @@ init_myhook(void)
 		eq_count_max = MAX_EQ;
 	}
 
+	/* D_IL_DAOS_MIN_FD=3 or smaller value will disable blocking low fds */
+	rc = d_getenv_uint64_t("D_IL_DAOS_MIN_FD", &daos_min_fd);
```

Comment:
is this env documented somewhere?

Reply:
It is not in our document. Normally users do not need to set it. Default value 10 will be used.

Reply:
ok well if users don't need to set it then i do not see why it needs to be an env variable..
if it needs to, then we should document it in the IL readme, and also consider changing it's name D_ and DAOS are repetitive.
maybe call it DAOS_RESERVE_MIN_FD? other suggestion welcome

Reply:
ok. We can just use a hard-coded value for now. We add an env later if necessary. Thank you!

Reply:
sorry, I am not against making this an env variable. im just saying that an env variable needs to be documented somewhere if we need to ask users to play with it. otherwise only the dev (you) knows about it and what it does.

Reply:
Thank you! Understood. We can use hardcoded value to make our code simpler and decrease the number of env users need to deal with. Originally, I thought bash script can use arbitrary fd. An env may be helpful if a larger value is needed. However, it could be very hard for a regular user to figure out the situation. I will add a warning in dup2() if a fd pointing to "socket:[xxxx]" or "anon_inode:[eventpoll]" is closed, so it would help for debugging related issues.